harfbuzz-sys/build.rs

```diff
@@ -44,5 +37,15 @@ fn main() {
     );
 }
 
-#[cfg(not(feature = "build-native-harfbuzz"))]
-fn main() {}
+fn main() {
+    println!("cargo:rerun-if-env-changed=HARFBUZZ_SYS_USE_PKG_CONFIG");
+    if env::var_os("HARFBUZZ_SYS_USE_PKG_CONFIG").map_or(false, |s| s != "0") {
+        // Force us to use pkg-config.
+        pkg_config::probe_library("harfbuzz").unwrap();
+    } else if cfg!(feature = "bundled") {
```

Comment:
I'd say it's probably fine to remove this environment variable now that this is controlled by a feature. Do you have a use case in mind where the feature isn't enough?

Reply:
This is for the case where someone naughty has enabled `bundled` and you want to force it to use the system ... it isn't needed for the usual scenario of bundled or not. It is just an escape hatch.

Reply:
I think we can remove it for now and then add it if this is really needed by a user.

Reply:
Removed for now.